### DISCUSSION TITLE
Upgrade to rc2.1 happens "The "AvaloniaStatsTask" task failed unexpectedly."

### DISCUSSION THREAD

**Discussion by jim-jiang-github** [AvaloniaUI/Avalonia#11987] [Q&A]
![image](https://github.com/AvaloniaUI/Avalonia/assets/29189410/cca3d1e3-3337-4629-93bc-8ff09f25bddd)

Is this a bug? Or I miss something?

**Comment by Mrxx99** [comment]
It is a bug, the Avalonia team is already working on it, fix should be release soon. In the meantime buildung and running from the commandline should work.

**Comment by danwalmsley** [comment]
https://www.nuget.org/packages/Avalonia/11.0.0-rc2.2

Fixed.
